utils/dyn-config.c

```diff
@@ -274,6 +276,7 @@ glusterBlockReadConfig(gbConfig *cfg, ssize_t *len)
   *len = 0;
   p = buf;
   while ((m = getline(&line, &n, fp)) != -1) {
+    empty = false;
```

Comment:
@lxbsz 
Also, why we need an extra variable here ?
Can we directly check if buf is empty ?

Reply:
@pkalever 
Yeah, fixed it.
Thanks.